Doc/library/codecs.rst

```diff
@@ -1443,10 +1443,26 @@ to :class:`bytes` mappings. They are not supported by :meth:`bytes.decode`
 | uu_codec             | uu               | Convert the operand using    |                              |
 |                      |                  | uuencode.                    |                              |
 +----------------------+------------------+------------------------------+------------------------------+
+| escape_codec         | escape           | Encode and decode byte       | :func:`codecs.escape_encode` |
+|                      |                  | sequences using escape       | /                            |
+|                      |                  | sequences, similar to        | :func:`codecs.escape_decode` |
+|                      |                  | :func:`repr` of bytes.       |                              |
++----------------------+------------------+------------------------------+------------------------------+
 | zlib_codec           | zip, zlib        | Compress the operand using   | :meth:`zlib.compress` /      |
 |                      |                  | gzip.                        | :meth:`zlib.decompress`      |
 +----------------------+------------------+------------------------------+------------------------------+
 
+.. function:: codecs.escape_encode(input, errors=None)
```

Comment:
Please move this section below the table and its notes.

It may be good to also be good to put these function definitions into a section "Standalone Codec Functions", or something like that, to make it clear that the functions exist without a codec (at the moment) and perhaps with an explanation why this is.